ports/mimxrt/boards/make-pins.py

```diff
@@ -235,6 +275,33 @@ def print_header(self, hdr_filename):
             hdr_file.write("extern const mp_obj_dict_t machine_pin_cpu_pins_locals_dict;\n")
             hdr_file.write("extern const mp_obj_dict_t machine_pin_board_pins_locals_dict;\n")
 
+            hdr_file.write("\n// Defines\n")
+            usdhcInstanceFactory(self.cpu_pins, hdr_file)
+
+
+def usdhcInstanceFactory(pins, output_file):
```

Comment:
this is a function so should be styled as `usdhc_instance_factory`

Reply:
done